src/components/form-item/form-item.jsx

```diff
@@ -1,187 +1,169 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 
 import './form-item.css';
 import ButtonClose from '../button-close';
 
-class FormItem extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      showSelectOption: 'invisible',
-    };
-    this.onClickSubmit = this.onClickSubmit.bind(this);
-    this.onClickReset = this.onClickReset.bind(this);
-    this.onClickSelectToggle = this.onClickSelectToggle.bind(this);
-  }
+const FormItem = (props) => {
+  const [showSelectOption, setShowSelectOption] = useState(false);
+  const onClickSelectToggle = () => setShowSelectOption(!showSelectOption);
```

Comment:
I'm not sure that we need such extra function to toggle selection
` onClick={() => setShowSelectOption(!showSelectOption)}` that's enough